pkg/store/tsdb_test.go

```diff
@@ -595,26 +580,6 @@ func TestTSDBStore_SeriesAccessWithDelegateClosing(t *testing.T) {
 		// Let's close pending querier!
 		testutil.Equals(t, 1, len(csrv.closers))
 		testutil.Ok(t, csrv.closers[0].Close())
-
-		// Expect flush and close to be unblocked and without errors.
```

Comment:
Why those are removed?

Reply:
hmmm prometheus/prometheus#8723 ... Can we skip them instead 🤔

Reply:
Yes, that's a better option.